lib/adapters/QUnitAdapter.js

```diff
@@ -5,18 +5,52 @@ import {Test, Suite} from "../Data.js";
 export default class QUnitAdapter extends EventEmitter {
     constructor() {
         super();
-        QUnit.done(this.onDone.bind(this));
+        QUnit.config.reorder = false;
```

Comment:
If we have reordering enabled, QUnit may emit multiple module done events for one suite. I think we don't want to have this in the standard?